lib.php

```diff
@@ -192,7 +192,15 @@ function learningmap_cm_info_view(cm_info $cm) : void {
     global $PAGE;
     // Only show map on course page if showdescription is set.
     if ($cm->showdescription == 1) {
-        $cm->set_content(learningmap_get_learningmap($cm), true);
+        $prepend = '';
+        if ($cm->groupmode > 0) {
+            $prepend = groups_print_course_menu(
+                $cm->course,
```

Comment:
I'm not 100% sure if this is the course object (which it should be) or just the course id. Safer would probably be to use the getter function `$cm->get_course()` here

Reply:
get_course() returns $this->course

Reply:
Well I don't think so, see:

https://github.com/moodle/moodle/blob/17ee0726933c7c6410aca0e1edc445f795774c07/lib/modinfolib.php#L1472

`$cm` is a `cm_info` object. If you access standard property course this will probably return the course id. Will have to doublecheck by testing

Reply:
Oh, that was course_modinfo (same file) - but I changed it already.